xrpl/models/requests/ledger_entry.py

```diff
@@ -152,6 +151,33 @@ class Ticket(BaseModel):
     """
 
 
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class XChainClaimID(BaseModel):
+    """Required fields for requesting an XChainClaimID if not querying by object ID."""
+
+    locking_chain_door: str
+    locking_chain_issue: Union[Literal["XRP"], IssuedCurrency]
```

Comment:
Can you turn this `Union[Literal["XRP"], IssuedCurrency]` into an amount for simplicity, or does that have negative side effects?

Reply:
Oh, nevermind - I see that XRP is a special value, not a string.